pyparsing/diagram/__init__.py

```diff
@@ -172,6 +229,9 @@ def mark_for_extraction(
     ):
         """
         Called when this instance has been seen twice, and thus should eventually be extracted into a sub-diagram
+        :param el_id:
```

Comment:
I guess in theory these should have descriptions

Reply:
Picky, picky. :) 

Will do.

Reply:
Yeah sorry this is a bit unnecessary. Normally for internal methods I just remove the parameter documentation entirely, I just don't like the look of a list of undocumented parameters I guess. But really it's up to you.

Reply:
No, perfectly valid. If I leave them out, PyCharm highlights the parameters saying "you forgot to document these in the docstring". So easier to add the description (as long as they are accurate).